[RTC-231] Add hints about using Protobufs (#10)
jellyfish-dev/jellyfish-docs: docs/api_reference.md

You can find them here:
* [OpenAPI REST description](https://github.com/jellyfish-dev/jellyfish/blob/main/openapi.yaml)
* [AsyncAPI client WS description](https://github.com/jellyfish-dev/jellyfish/blob/main/docs/jellyfish-ws.yaml)
* [Protobufs used by server WS](https://github.com/jellyfish-dev/protos/blob/RTC-231-move-server-notifications-to-protobuffs/jellyfish/server_notifications.proto)

The communication between peer WS and server is done with use of JSON format.
The very first message that has to be sent on peer WS is
authorization request.

The communication from server to peer WS is done with use of Protobuf format.
All supported messages are listed in [the Jellyfish protos repo](https://github.com/jellyfish-dev/protos).
If you want to create your own Jellyfish SDK, the easiest way to reference to defined protobuf messages will be by adding protos to your git repository as a git submodule. You can do it with this command:
```bash
git submodule add https://github.com/jellyfish-dev/protos
```

> You could also consider creating a script that will pull newest changes from the git submodule and compile proto files to your chosen programming language. For reference, see [the script used in our Elixir SDK](https://github.com/jellyfish-dev/elixir_server_sdk/blob/RTC-231-move-server-notifications-to-protobuffs/compile_proto.sh).

Reviewer: Please put empty lines around code blocks.

    console.log("Message from server ", event.data);
});
ws.send(authRequest)
```

> The communication between peer WS and server is done with use of Protobuf format.

Reviewer 2: It's the same sentence as in line 13, only with `Protobuf` instead of `JSON`, which can be confusing. We should probably clarify that we're referring to server notifications here, maybe something along the lines of "communication *from* server *to* peer WS".

> All supported messages are listed in [jellyfish protos repo](https://github.com/jellyfish-dev/protos).
> If you want to create your own jellyfish SDK the easiest way to reference to defined protobuf messages will be by adding to your git repository protos as git submodule. You can do this with this command:

Reviewer 2: ```suggestion
All supported messages are listed in [the Jellyfish protos repo](https://github.com/jellyfish-dev/protos).
If you want to create your own Jellyfish SDK, the easiest way to reference to defined protobuf messages will be by adding protos to your git repository as a git submodule. You can do it with this command:
```

```bash
git submodule add https://github.com/jellyfish-dev/protos
```

> Also you could think about creating script that will pull newest changes from git submodule and also compile proto files to your chosen programming language, [example script used in our elixir sdk](https://github.com/jellyfish-dev/elixir_server_sdk/blob/RTC-231-move-server-notifications-to-protobuffs/compile_proto.sh). 

Reviewer 2: ```suggestion
You could also consider creating a script that will pull newest changes from the git submodule and compile proto files to your chosen programming language. For reference, see [the script used in our Elixir SDK](https://github.com/jellyfish-dev/elixir_server_sdk/blob/RTC-231-move-server-notifications-to-protobuffs/compile_proto.sh).
```
